### DISCUSSION TITLE
Remote SSH Support for Non-Enterprise

### DISCUSSION THREAD

**Discussion by kaaax0815** [coder/code-server#2612] [Q&A]
On your website you featured the Remote SSH Tool on VSCode locally, I didnt get it to work, is this a problem from me, or is this not supported by the normal version?

**Comment by code-asher** [comment]
Ah I think that's for accessing Coder's cloud from local/desktop VS Code
using the remote extensions. It's not related to code-server in any way.

Since those extensions are designed to only work on VS Code we have an
issue for implementing our own for code-server: https://github.com/cdr/code-server/issues/1315
